po/common/common.pot

```diff
@@ -662,7 +662,7 @@ msgid "The user name must contain only unaccented letters, digits and dashes."
 msgstr ""
 
 msgctxt "error_username_too_long"
-msgid "The user name is too long (maximum 20 characters)."
+msgid "The user name is too long (maximum 40 characters)."
```

Comment:
FYI @benbenben2, you don't have to change this in every file. Changing it in common.pot it enough (crowdin handles the rest).